oneanddone/users/views.py

```diff
@@ -162,7 +162,7 @@ class UserDetailAPI(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = UserSerializer
 
 
-class UserListAPI(generics.ListCreateAPIView):
+class UserListAPI(generics.CreateAPIView):
```

Comment:
As this is being changed from a class that lists users, to one that just creates users, you should update the name accordingly.